wkhtmltopdf/views.py

```diff
@@ -65,47 +60,6 @@ def __init__(self, request, template, context=None,
             cmd_options = {}
         self.cmd_options = cmd_options
 
-    def render_to_temporary_file(self, template_name, mode='w+b', bufsize=-1,
```

Comment:
As this method has been removed from  `PDFResponse` we need to make sure it is recorded as a backwardly incompatible change.